testsuite/features/step_definitions/common_steps.rb

```diff
@@ -752,7 +752,10 @@
     puts $minion.run("zypper mr --disable #{repos}")
   end
 
-  $minion.run('zypper -n --gpg-auto-import-keys ref')
+  # Refresh is only necessary when some repos are enabled
+  if $minion.run('zypper lr').contains? 'Yes'
+    $minion.run('zypper -n --gpg-auto-import-keys ref')
```

Comment:
I guess his only needed when the client tools come from D:G:M:* projects?

Otherwise, why would we need to do the import and the refresh manually?

Reply:
Yep, I understand it the same way.